ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -593,11 +583,13 @@
                   combined-trips)
         ;; When dealing with new routes there aren't traffic at date1-trips because traffic is starting
         ;; So calculate only new trips, no other changes or stops
-        added-trip-count (when (and (nil? combined-trips) (pos-int? (count date2-trips)))
-                      (count date2-trips))
+        added-trip-count (if (and (nil? combined-trips) (pos-int? (count date2-trips)))
+                      (count date2-trips)
+                      0)
         ;; When traffic is ending there isn't traffic at date2-trips vector. So calculate only ending trips.
-        removed-trip-count (when (and (nil? combined-trips) (pos-int? (count date1-trips)))
-                           (count date1-trips))]
+        removed-trip-count (if (and (nil? combined-trips) (pos-int? (count date1-trips)))
+                           (count date1-trips)
```

Comment:
I had to add this zero to ui because it was missing. Nice work.
Check intendation